src/app/_services/flag.service.ts

```diff
@@ -35,26 +35,20 @@ export class FlagService {
     ).pipe(
     map(res => {
       let flags = res.body
-      let userCountryCode: string = res.headers.get('x-artstor-country-code').substr(0, 2)
 
-      /**
-       * EXAMPLE USE OF COUNTRY CODE ROLLOUT
-       * // Boolean assignments
-       * this.unaffiliated = flags.unaffiliatedAccess
-       *
-       * // If the user's country code is allowed, set unaffiliated flag to true
-       * let userCountryCode: string = res.headers.get('x-artstor-country-code').substr(0, 2)
-       * if (flags.unaffiliatedAccessRollout.indexOf(userCountryCode) > -1) {
-       *  this.unaffiliated = true
-       * }
-       */
```

Comment:
We should leave the commented example in, for future use? Makes clean up later easier